packages/mds-db/migration.ts

```diff
@@ -181,7 +181,7 @@ async function dropReadOnlyGeographyColumnMigration(exec: SqlExecuterFunction) {
 async function dropAuditEventsColumnsMigration(exec: SqlExecuterFunction) {
   await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN provider_event_id`)
   await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN provider_event_type`)
-  await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN provider_event_type_reason`)
+  // await exec(`ALTER TABLE ${schema.TABLE.audit_events} DROP COLUMN provider_event_type_reason`)
```

Comment:
Will we also need another migration to drop the `provider_event_type_reason` column in prod?

Reply:
yep! i got a separate branch.